src/lib/pmd/PmdEngine.ts

```diff
@@ -4,10 +4,12 @@ import {Controller} from '../../Controller';
 import {Catalog, Rule, RuleGroup, RuleResult, RuleTarget} from '../../types';
 import {RuleEngine} from '../services/RuleEngine';
 import {Config} from '../util/Config';
-import {ENGINE} from '../../Constants';
+import {ENGINE, CUSTOM_CONFIG, EngineFlavor} from '../../Constants';
 import {PmdCatalogWrapper} from './PmdCatalogWrapper';
 import PmdWrapper from './PmdWrapper';
 import {uxEvents} from "../ScannerEvents";
+import { FileHandler } from '../util/FileHandler';
+import { EventCreator } from '../util/EventCreator';
```

Comment:
It seems like this is intended to replace uxEvents, but I still see that being used in places. Am I misunderstanding?